documentation/guide-jpa.asciidoc

```diff
@@ -369,11 +369,12 @@ We strongly recommend these principles:
 * Keep your entities simple and avoid putting advanced logic into entity methods.
 
 == Database Configuration
-The link:guide-configuration.asciidoc[configuration] for spring and hibernate is already provided by devonfw in our sample application and the application template. So you only need to worry about a few things to customize.
+The link:guide-configuration.asciidoc[configuration] for spring and Hibernate is already provided by devonfw in our sample application and the application template. So you only need to worry about a few things to customize. In Quarkus, Hibernate is provided by the `quarkus-hibernate-orm` extension.
 
 === Database System and Access
-Obviously you need to configure which type of database you want to use as well as the location and credentials to access it. The defaults are configured in `application-default.properties` that is bundled and deployed with the release of the software. It should therefore contain the properties as in the given example:
+Obviously you need to configure which type of database you want to use as well as the location and credentials to access it. The defaults are configured in `application-default.properties` that is bundled and deployed with the release of the software. In Quarkus applications, use the `applications.properties` file to make the configurations. The files should therefore contain the properties as in the given example:
```

Comment:
I do not know where this `application-default.properties` actually comes from but this is wrong.
Both for spring and for quarkus it is `application.properties`.
IMHO this entire "Configuration" sub-section needs a face-lift but fine for now and can be improved also in a later PR to make it simple.